src/content/reference/gsctl/update-org-set-credentials.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: "gsctl Command Reference: update organization set-credentials"
 description: The 'gsctl update organization set-credentials' command allows to set cloud provider credentials for an organization.
-date: 2018-09-12
+date: 2020-05-19
 type: page
 weight: 55
 ---
 
 # `update organization set-credentials`
 
-Giant Swarm allows you to run clusters in your own cloud provider account/subscription. We call this by the catchy term [Bring Your Own Cloud](/basics/byoc/) (BYOC). As a prerequisite, the organizaiton that should own the clusters has to be prepared with cloud provider credentials.
+Giant Swarm allows you to run clusters in your own cloud provider account/subscription. We refer to this as [Multi Account functionality](/basics/byoc/), renamed from Bring Your Own Cloud (BYOC). As a prerequisite, the organizaiton that should own the clusters has to be prepared with cloud provider credentials.
```

Comment:
are we not changing the link? `/byoc/`

Reply:
```suggestion
Giant Swarm allows you to run clusters in your own cloud provider account/subscription. We refer to this as [Multi Account functionality](/basics/multi-account/), renamed from Bring Your Own Cloud (BYOC). As a prerequisite, the organization that should own the clusters has to be prepared with cloud provider credentials.
```

Reply:
btw, did you rename that already? need to merge the changes together

Reply:
I did rename it [PR #506](https://github.com/giantswarm/docs/pull/506)